packages/assets-controllers/src/NftController.test.ts

```diff
@@ -114,14 +123,17 @@ function setupController({
   return {
     assetsContract,
     nftController,
-    network,
     onNftAddedSpy,
     preferences,
     messenger,
+    changeNetwork,
   };
 }
 
 describe('NftController', () => {
+  const sepolia = { chainId: '11155111', type: 'sepolia' as NetworkType };
```

Comment:
Instead of keeping these at the top of the `describe`, what do you think about keeping these at the top of the file along with the other `const`s?

Also instead of `as NetworkType` what do you think about `as const`? The first is a typecast, but the second is a type inference.

Reply:
`as const` doesn't work, but Ill move it next to the other const's and all caps it.

Reply:
my bad, it does work I just did it wrong lol. fixing.